### DISCUSSION TITLE
脚本 ./src 下的关于动态导入有没有什么约定？

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#3819] [提问 / Q&A] [answered]
在 ./src 的代码中，有许多地方使用了动态导入。猜测应该是为了避免依赖循环。不知道这里面有没有什么约定，比如在什么地方引入什么应该使用动态导入。或者默认静态导入，等编译器报错时才改成动态？

**Comment by the1812** [comment, marked as answer]
就是依赖循环, 能动态尽量动态
